h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -990,6 +992,15 @@ public void init(boolean expensive) {
         error(_parms._non_negative ? "non_negative" : "beta_constraints",
                 " does not work with " + _parms._family + " family.");
       }
+      // maximum likelhood is only allowed for families tweedie, gamma and negativebinomial
+      if (ml.equals(_parms._dispersion_factor_mode) && !gamma.equals(_parms._family))
```

Comment:
should we call this "mode" or rename to "method" (I know this is just nitpicking but "method" seems more natural fit and "mode" has a meaning in statistics)

Reply:
Changed to method.